tests/misc_env/install_prereq.py

```diff
@@ -65,6 +66,7 @@ def install_prereq(ceph, timeout=1800, skip_subscription=False, repo=False, rhbu
             ceph.exec_command(cmd='sudo pip install crefi', long_running=True)
         ceph.exec_command(cmd='sudo yum clean metadata')
         config_ntp(ceph)
+    registry_login(ceph, distro_ver)
```

Comment:
did you check negative cases ? if other suites fail or pass if the cdn_credentials vars are not provided or updated  cephci.yaml.template doesnt exists?

Reply:
@XDsubhash Yes I have tested negative cases, if cdn_credentials not provided in cephci.yaml file then install_prereq will fail, I think It's good to have this as compulsory task in prereq so that other suites can perform cdn container testing with this logged in registry also this logged in registry doesn't affect with distro registry i.e registry-proxy.engineering.redhat.com
we just have to update everyone to provide these credentials before running and they can trigger any distro or any container cdn suite

Reply:
the failure should not come for every execution. the failure should be seen only if its a container suite . why would we this if we are not running non-container tests without dashboard

Reply:
@rakeshgm for dashboard both bare-metal and container, we should be doing login to 'registry.redhat.io' so i made it as compulsory task in prereq since this is not required for non-container tests without dashboard, I will add some checks not to fail for rpm based tests without cdn credentials, Thanks for pointing it out